packages/subgraph/src/mappings/superToken.ts

```diff
@@ -447,3 +450,14 @@ function _createTransferEventEntity(event: Transfer): void {
     ev.token = event.address;
     ev.save();
 }
+
+export function handleApproval(event: Approval): void {
+    const flowOperatorID = getFlowOperatorID(event.params.spender, event.address, event.params.owner);
```

Comment:
I would call `getOrInitFlowOperator` here and also create a new `ApprovalEvent` entity maybe.